include/rapidjson/schema.h

```diff
@@ -578,7 +580,9 @@ class Schema {
     }
 
     ~Schema() {
-        allocator_->Free(enum_);
+        if (allocator_) {
```

Comment:
allocator_ is NULL for the Schema returned by Schema::GetTypeless(). The destructor is dereferencing the NULL when the global static is torn down during shutdown.